TextStyling-Kotlin/app/src/main/java/com/android/example/text/styling/renderer/spans/BulletPointSpan.kt

```diff
@@ -45,49 +46,51 @@ class BulletPointSpan(
      * margins before the bullet.
      */
     override fun drawLeadingMargin(
-            c: Canvas, p: Paint, x: Int, dir: Int, top: Int, baseline: Int, bottom: Int,
-            text: CharSequence, start: Int, end: Int, first: Boolean, l: Layout
+        c: Canvas, p: Paint, x: Int, dir: Int, top: Int, baseline: Int, bottom: Int,
```

Comment:
would you mind renaming `c` to `canvas` and `p` to `paint`, please? it would make the code more readable.